README.md

```diff
@@ -1,10 +1,13 @@
 # Coding Test
 
-In order to gain insights in your development skills, we created this development exercise.
+Do you want to join the engineering team at [Teamleader](https://www.teamleader.eu/company/engineering)?
+
+In order to gain insights in your development skills, we have created this exercise.
```

Comment:
PP: Invert the sentence. Also, I think it should be "into your development skills":

We created this exercise in order to gain insights into your development skills.

Reply:
Changing!